lib/Mojolicious/Command/daemon.pm

```diff
@@ -18,10 +18,13 @@ sub run {
     'i|inactivity-timeout=i' => sub { $daemon->inactivity_timeout($_[1]) },
     'k|keep-alive-timeout=i' => sub { $daemon->keep_alive_timeout($_[1]) },
     'l|listen=s'             => \my @listen,
-    'p|proxy'                => sub { $daemon->reverse_proxy(1) },
+    'p|proxy:s'              => \my @proxy,
     'r|requests=i'           => sub { $daemon->max_requests($_[1]) };
 
   $daemon->listen(\@listen) if @listen;
+  $daemon->reverse_proxy(1) if @proxy;
+  my @trusted = grep {length} @proxy;
+  $daemon->trusted_proxies(\@trusted) if @trusted;
```

Comment:
Pretty sure this can be done with less code too.

Reply:
How about just `$daemon->trusted_proxies([grep {length} @proxy])` ?